packages/wonder-blocks-announcer/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@khanacademy/wonder-blocks-announcer",
+  "version": "0.0.1",
+  "design": "v1",
+  "description": "Live Region Announcer for Wonder Blocks.",
+  "main": "dist/index.js",
+  "module": "dist/es/index.js",
+  "source": "src/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "types": "dist/index.d.ts",
+  "author": "",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@khanacademy/wonder-blocks-core": "^7.0.1"
```

Comment:
**suggestion:** You don't need to include `core` as dependency as it seems that it is not going to be used in the bundle. If this is for testing and/or storybook, you can include it as a `devDependency`.